smdebug_rulesconfig/debugger_rules/_ruleGroups.py

```diff
@@ -1,29 +1,30 @@
-from enum import Enum
-
-from ._utils import _get_config_for_group
-
-
-class MXNET(Enum):
-    SIMPLE = ["VanishingGradient", "LossNotDecreasing", "WeightUpdateRatio"]
-    ALL = []
-
-
-class TENSORFLOW(Enum):
-    SIMPLE = ["VanishingGradient", "LossNotDecreasing", "WeightUpdateRatio"]
-    ALL = []
-
-
-class PYTORCH(Enum):
-    SIMPLE = ["VanishingGradient", "LossNotDecreasing", "WeightUpdateRatio"]
-    ALL = []
-
-
-class XGBOOST(Enum):
-    SIMPLE = ["TreeDepth", "ClassImbalance"]
-    ALL = []
-
-
-def get_rule_groups(ruleEnum):
-    ruleEnumVal = ruleEnum.value
-    rules_config = _get_config_for_group(ruleEnumVal)
-    return rules_config
+# set of rules that are expected to work for all supported frameworks
+# Supported Frameworks: Mxnet, Pytorch, Tensorflow, Xgboost
+UNIVERSAL_RULES = {
+    "AllZero",
+    "ClassImbalance",
+    "LossNotDecreasing",
+    "Overfit",
+    "Overtraining",
+    "SimilarAcrossRuns",
+    "StalledTrainingRule",
+    "TensorVariance",
```

Comment:
TensorVariance rule is not applicable to XGBoost models.

Reply:
Thanks. We'll need to update the docs as well: https://docs.aws.amazon.com/sagemaker/latest/dg/debugger-built-in-rules.html